8.x/theme-development/client-side-cart-module.md

```diff
@@ -0,0 +1,52 @@
+## Client Side cart module
+
+[[toc]]
+
+The Larammerce Template Engine has the **local card module**. In this tutorial, the source code of this module is Reviewed.
```

Comment:
Larammerce base theme has a build-in JS module named `LocalCartService` to manage the client-side cart in the customer's browser.
This document reviews the life cycle and progress of this module.